pwa/src/components/logs/logModal/LogModal.tsx

```diff
@@ -100,19 +100,35 @@ const LogModal: React.FC<LogModalProps> = ({ log }) => {
                       {
                         title: "Session",
                         id: "logRequestSession",
-                        backgroundColor: "black",
-                        render: function () {
+                        render: function() {
                           return (
                             <>
                               {log.sessionValues ? (
-                                <CodeBlock code={JSON.stringify(log.sessionValues)} language="json" />
+                                <table className="mt-3 logTable-table">
+                                  <tr>
+                                    <th>Active Organization</th>
+                                    <td>{log.sessionValues.activeOrganization}</td>
+                                  </tr>
+                                  <tr>
+                                    <th>Parent Organizations</th>
+                                    <td>{log.sessionValues.parentOrganizations[0]}</td>
+                                  </tr>
+                                  <tr>
+                                    <th>Organizations</th>
+                                    <td>{log.sessionValues.organizations[0]}</td>
+                                  </tr>
+                                  <tr>
+                                    <th>Application</th>
+                                    <td>{log.sessionValues.application.name}</td>
+                                  </tr>
+                                </table>
```

Comment:
(I think the following feedback will also resolve your comment about having to show all content in the description of this pull request)

Right now, you're hardcoding the rows (_Active organisation, Parten organisation, Organisations_ and _Application_). This works fine for these rows, however, when a session has more values it will never show them (e.g.  `callId`, `handler`, `endpoint`, ...).

A better solution might be (the same as we implemented in the _Headers_ accordions:

```
const logs = [];
for (const [key, value] of Object.entries(log.requestHeaders)) {
  logs.push({ ...{ key, value } });
}

// at this point we have a mappable array (logs) with simpele {log} entries
// so now we're able to map over the logs and render them, like so:

{logs.map((log, idx) => {
  return (
    <tr key={log.key + idx}>
    <th>{log.key}</th>
    <td>{log.value}</td>
    </tr>
   );
})}
```

Note: some of these `{ log.value }` entries are quite big (they are code objects themselves). So it might be best to show them as `CodeBlocks`. Maybe you can verify this with @rubenvdlinde. I think just rendering these big objects or arrays like plain text wouldn't be a good idea.